packages/framework-provider-kubernetes-infrastructure/src/infrastructure/index.ts

```diff
@@ -4,18 +4,25 @@ import { HelmManager } from './helm-manager'
 import { DeployManager } from './deploy-manager'
 import { DaprManager } from './dapr-manager'
 import { Promises } from '../helpers/promises'
+export interface BoosterK8sConfiguration extends BoosterConfig {
```

Comment:
Any suggestion here to add a new property to BoosterConfig without the need of extends and create a new interface. I don't really like this approach.

Reply:
That, in fact, is the way to adding a new property to BoosterConfig 😂